src/main/java/software/amazon/documentdb/jdbc/DocumentDbConnection.java

```diff
@@ -161,9 +116,10 @@ public int getSshLocalPort() {
      *
      * @return returns {@code true} if the SSH tunnel is active, {@code false}, otherwise.
      */
+    @SneakyThrows
     public boolean isSshTunnelActive() {
         // indicate whether the SSH tunnel is enabled
-        return session != null;
+        return sshTunnelClient != null && sshTunnelClient.getServiceListeningPort() > 0;
```

Comment:
Shall we add a troubleshooting section, in case SSH tunnel connection is reset or broken? I believe we won't be able to handle this case in the code.

Reply:
Added a new section in src/markdown/support/troubleshooting-guide.md

Reply:
👍